db/view/base_info.hh

```diff
@@ -16,20 +16,8 @@ namespace db {
 
 namespace view {
 
```

Comment:
> view_info: remove base schema from the base_info

If I understand correctly, view_info now represents the SELECT statement (plus any parts of the base schema that were made invariant by the view's existence). So view_info is now a sort of operator, that you can apply to the base schema, and get a derived view_schema.

Reply:
Right, you can treat it that way

Reply:
I understood the comment as referring to the `view_info` even though it was placed inline in `base_info.hh`. I suspect it was placed there because it was actually referring to the commit message, and there's no good place in github to comment on it